src/site/twitch.tv/modules/emote-menu/EmoteMenuSet.vue

```diff
@@ -90,6 +90,10 @@ function sortCase(ae: SevenTV.ActiveEmote): number {
 const filterEmotes = debounceFn((filter = "") => {
 	const x = [] as SevenTV.ActiveEmote[];
 
+	if (props.es.provider && props.es.emotes.length === 0) {
```

Comment:
nit: this can be simplified

```suggestion
	if (props.es.provider && !props.es.emotes.length) {
```

Reply:
@AnatoleAM PR updated and nit fixed.